### PR TITLE
new Jindra

### PR DESCRIPTION
**Pull request by jindrichsamec**
_<<< Description here >>>_

#### Checklist
- [x] Unit Tests (where helpful and possible)
- [ ] Gherkin Tests **OR** Test Scenarios in TP (https://brandembassy.tpondemand.com/entity/1543)
- [ ] Accepted by PO (only for features or when cutting features, not for bugs)

### CODE REVIEW

File: prvni.txt [hunk 1]
```diff
@@ -1,2 +1,8 @@
 prvni radek
 druhy radek
+
+
+treti
+4.
```
Comment (jindrichsamec) [inline prvni.txt]: tady nesouhlasím s číslovkou!

File: prvni.txt [hunk 2]
```diff
@@ -1,2 +1,8 @@
...
+
+pátý
```
Comment (jankrejcik) [inline prvni.txt]: Muhehe

File: prvni.txt (hunk 1 above)
Comment (martinmares) [reply prvni.txt]: je to ok

Comment (pavekron) [reply prvni.txt]: me se libi

**Pull request closed without merging**